hosts/build02/default.nix

```diff
@@ -11,6 +11,8 @@
     inputs.srvos.nixosModules.hardware-hetzner-online-amd
   ];
 
+  nix.settings.auto-optimise-store = lib.mkForce false;
```

Comment:
My experience has been that is keep-derivations that slows down gc quite a bit:

sudo nix-collect-garbage -d --option keep-derivations false

Reply:
I'll let it fill up and try that.

Reply:
`keep-derivations false` helped a little but still very slow, I forgot to copy the exact gc output but hardlinking was saving a lot of space. I thought I'd reinstall as it would be quicker than gc. nixos-anywhere is now very fast, only took about five minutes.